setup.cfg

```diff
@@ -75,7 +75,7 @@ torch =
     torch>=1.0.0,<1.4.0
 tensorflow =
     tensorflow>=2.0.0,<2.1.0
-    tfdlpack>=0.1.1,<0.2.0
+    tfdlpack>=0.1.1,<0.2.0; python_version == "3.7"
```

Comment:
Maybe we should abandon the tfdlpack thing altogether?

Like, it only installs on Linux on particular Python versions, it requires tricky pip installation, it's unclear whether it'll be maintained well in the future, and it's not even obvious it will make a big performance difference.